Generic_Plugin_Admin.php

```diff
@@ -409,25 +426,34 @@ function admin_print_styles() {
 	}
 
 	/**
-	 * Print scripts
-	 *
-	 * @return void
+	 * Print scripts.
 	 */
 	function admin_print_scripts() {
 		wp_enqueue_script( 'w3tc-metadata' );
 		wp_enqueue_script( 'w3tc-options' );
 		wp_enqueue_script( 'w3tc-lightbox' );
 
 		if ( $this->is_w3tc_page ) {
-			wp_localize_script( 'w3tc-options', 'w3tc_nonce',
-				array( wp_create_nonce( 'w3tc' ) ) );
+			wp_localize_script(
+				'w3tc-options',
+				'w3tc_nonce',
+				array( wp_create_nonce( 'w3tc' ) )
+			);
 		}
 
-
 		switch ( $this->_page ) {
 		case 'w3tc_minify':
-		case 'w3tc_mobile':
-		case 'w3tc_referrer':
+		case 'w3tc_cachegroups':
+			wp_enqueue_script(
```

Comment:
the same here.
js for specific pages are loaded in plugin code itself for later code.